### PR TITLE
Issue#1165 tests fix

### PR DESCRIPTION
**Pull request by yugander-krishan-singh**
Fixes for [#1165](https://github.com/strongbox/strongbox/issues/1165):

- [x] ArtifactManagementServiceImplTest

### CODE REVIEW

**Comment by yugander-krishan-singh** [comment]
@carlspring the checks are failing due to some tests failures which are not contributed by me. Pls let me know if anything is required from my side.

**Comment by yugander-krishan-singh** [comment]
@carlspring please let me know if this PR is even required. 
Thanks.

File: strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactManagementServiceImplTest.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
 import java.util.Set;
 import java.util.concurrent.Callable;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
```
Comment (carlspring) [inline strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactManagementServiceImplTest.java]: These three imports (above) are probably not required.

**Comment by carlspring** [comment]
@yugander-krishan-singh ,

Apologies for the late reply, but I have been a little busy! Would you mind rebasing and fixing the handful of things I raised, so that we could merge this? Thanks! :)

**Comment by carlspring** [comment]
@yugander-krishan-singh : Ping? :)

**Comment by yugander-krishan-singh** [comment]
@carlspring sry for not replying earlier. I was not well and am taking medications. I will fix the things you mentioned and will also rebase in few hours.

Thanks

**Comment by carlspring** [comment]
Oh, I hope you feel better soon!

I was just making sure this pull wasn't getting abandoned.

Thanks!

File: strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactManagementServiceImplTest.java (hunk 1 above)
Comment (yugander-krishan-singh) [reply strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactManagementServiceImplTest.java]: AtomicBoolean import is required as it was being used. I have removed the other two.

**Comment by yugander-krishan-singh** [comment]
@carlspring I have rebased the code and also made the changes as suggested in the comments.
Pls let me know if anything else is required from my side.

Thanks

**Comment by carlspring** [comment]
Thanks for your help, @yugander-krishan-singh! :)

**Pull request merged**